src/ngraph/frontend/onnx_import/op/sum.hpp

```diff
@@ -18,7 +18,8 @@
 
 #include "core/node.hpp"
 #include "ngraph/node.hpp"
-#include "ngraph/op/add.hpp"
+#include "ngraph/opsets/opset0.hpp"
+#include "ngraph/opsets/opset1.hpp"
```

Comment:
This is example how we can use different opsets within one file. 
Rather by explicit opsets includes than `default_opset.hpp`.